Core/TermsOfService.swift

```diff
@@ -23,9 +23,24 @@ import Foundation
 
 public struct TermsOfService {
 
-    public let classification: Classification?
-    public let score: Int
-    
+    private static let classificationScores = [
```

Comment:
Optional, reformat to  more common enum `.`  reference e.g:
```
private static let classificationScores: [Classification: Int] = [
        .a: -1,
        .b: 0,
        .c: 0,
        .d: 1,
        .e: 2
    ]
```

Reply:
Yeah I agree, that's nicer.